### PR TITLE
Add score searching function

### PR DESCRIPTION
**Pull request by idujin**

### CODE REVIEW

File: csv_editor.py [hunk 1]
```diff
@@ -85,18 +85,20 @@ def _update_csv(self, today: datetime, user:str, time:str, excuse:str, score:int
             df.iloc[-1] = new_log
             df.to_csv(file_name, header =True, mode='w')
 
-    def get_display_csv(self, user):
+    def get_display_csv(self, user, year: int=0, month: int=0):
```
Comment (yhbyhb) [inline csv_editor.py]: use date time.now() for year and month rather than 0. same as below other functions

File: dateman.py [hunk 2]
```diff
@@ -24,7 +24,11 @@ def __init__(self, date_to_key: str):
         self.url_base = DATE_URL
         self.holidays_frame = None
         self.last_updated ="2023-01-01"
-
+    def get_current_year(self):
```
Comment (yhbyhb) [inline dateman.py]: I'm not sure these are necessary. Isn't date time.today() enough?

**Review by yhbyhb — APPROVED** [review]

File: csv_editor.py (hunk 1 above)
Comment (idujin) [reply csv_editor.py]: Done!

File: dateman.py (hunk 2 above)
Comment (idujin) [reply dateman.py]: Done

**Pull request merged**